src/Jackalope/ObjectManager.php

```diff
@@ -869,6 +869,7 @@ public function save()
                 if ($item->isModified() || $item->isMoved()) {
                     $item->confirmSaved();
                 }
+                $item->setClean();
```

Comment:
Not sure I understand: do you mean if the node contains some auto-created properties? i.e. if it is dirty then the state will be reloaded somehow?

Otherwise do you think this is the correct behavior? If not, any ideas as to what we can do to fix the issue?